src/library/analysisfeature.cpp

```diff
@@ -86,20 +86,34 @@ void AnalysisFeature::bindWidget(WLibrary* libraryWidget,
     m_pAnalysisView = new DlgAnalysis(libraryWidget,
                                       m_pConfig,
                                       m_library);
-    connect(m_pAnalysisView, SIGNAL(loadTrack(TrackPointer)),
-            this, SIGNAL(loadTrack(TrackPointer)));
-    connect(m_pAnalysisView, SIGNAL(loadTrackToPlayer(TrackPointer, QString)),
-            this, SIGNAL(loadTrackToPlayer(TrackPointer, QString)));
-    connect(m_pAnalysisView, SIGNAL(analyzeTracks(QList<TrackId>)),
-            this, SLOT(analyzeTracks(QList<TrackId>)));
-    connect(m_pAnalysisView, SIGNAL(stopAnalysis()),
-            this, SLOT(stopAnalysis()));
-
-    connect(m_pAnalysisView, SIGNAL(trackSelected(TrackPointer)),
-            this, SIGNAL(trackSelected(TrackPointer)));
-
-    connect(this, SIGNAL(analysisActive(bool)),
-            m_pAnalysisView, SLOT(slotAnalysisActive(bool)));
+    connect(m_pAnalysisView,
+            &DlgAnalysis::loadTrack,
+            this,
+            &AnalysisFeature::loadTrack);
+    connect(m_pAnalysisView,
+            &DlgAnalysis::loadTrackToPlayer,
+            this,
+            [&](TrackPointer track, QString group) {
```

Comment:
- The `this` pointer can be passed by value so `[=]` would be safer ...I don't trust the C++ compiler when using `[&]` ;)
- `this->` could be omitted, I'm unsure about best practices here
- I prefer to use `emit` without braces in new code, because it is a special keyword and not a simple function call. Not sure what the community prefers.